frontend/public/style/_common.scss

```diff
@@ -405,6 +405,11 @@ $co-external-link-padding-right: 15px;
   white-space: nowrap;
 }
 
+.co-wrap {
```

Comment:
Use the existing `co-break-word` and remove this.

Reply:
I changed to `<div>`'s and used `co-break-word`, but the table headers titles do not wrap using these.